src/WebJobs.Script.WebHost/Standby/StandbyManager.cs

```diff
@@ -107,13 +107,13 @@ public async Task SpecializeHostCoreAsync()
             // user dependencies
             FunctionAssemblyLoadContext.ResetSharedContext();
 
+            NotifyChange();
```

Comment:
please add a comment why NotifyChange needs to be called before specializing language workers